app/src/main/java/com/gxwtech/roundtrip2/RoundtripService/RoundtripService.java

```diff
@@ -86,11 +87,17 @@ public void onCreate() {
         mContext = getApplicationContext();
         serviceConnection = new RoundtripServiceIPCConnection(mContext);
 
-        sharedPref = mContext.getSharedPreferences(RT2Const.serviceLocal.sharedPreferencesKey, Context.MODE_PRIVATE);
```

Comment:
Is this cool? From what I have read there is no about Context.MODE_PRIVATE there is no difference using using it or not